app/package.json

```diff
@@ -61,15 +58,16 @@
     "pretty-bytes": "^6.1.1",
     "psl": "https://github.com/lupomontero/psl",
     "svelte-collections": "^1.2.4",
-    "svelte-i18n-lingui": "^0.1.7",
     "svelte-icon": "^2.0.0",
+    "svelte-icu-l10n": "https://github.com/mustang-im/svelte-icu-l10n",
```

Comment:
As mentioned in chat, please put the functions directly into our l10n.ts.

Please simplify the functions as well. Use as much of our existing functions as possible, then let the new functions do the minimal amount of work to do the rest.

Please document the new functions, to say *why* they are needed. (Don't describe *what* they do, but *why* it's needed.)